src/utils/thread_helper.cpp

```diff
@@ -920,6 +920,42 @@ void ThreadHelper::DetachGracefullyCallback(void)
 }
 
 #if OTBR_ENABLE_TELEMETRY_DATA_API
+#if OTBR_ENABLE_BORDER_ROUTING
+void ThreadHelper::GetNodeDivergenceInfo(threadnetwork::TelemetryData_NodeDivergenceInfo *aNodeDivergenceInfo)
+{
+    bool isExternalRoutePublished = false;
+    bool isDefaultRoutePublished  = false;
+    Ip6Prefix prefix;
+    uint16_t rloc16               = otThreadGetRloc16(mInstance);
+
+    otNetworkDataIterator iterator = OT_NETWORK_DATA_ITERATOR_INIT;
+    otExternalRouteConfig config;
+
+    // iterate over leader network data
```

Comment:
iterate => Iterate. Correct the following comment's first word as well.

Reply:
updated! Thanks!